dags/surveymonkey_data_import_pipeline.py

```diff
@@ -131,23 +130,19 @@ def surveymonkey_survey_answers_etl(**kwargs):
     for survey_id_list in survey_id_list_from_df:
         survey_id = str(survey_id_list[1])
         LOGGER.info("answers for survey_id: %s", survey_id)
-        list_of_answers_of_one_survey = get_survey_answer_details(
+        iterable_of_answers_of_one_survey = iter_formated_survey_user_answers(
             access_token=get_surveymonkey_access_token(),
             survey_id=survey_id
         )
-        LOGGER.info("length of answers is %s", len(list_of_answers_of_one_survey))
-        if not list_of_answers_of_one_survey:
+        if not iterable_of_answers_of_one_survey:
```

Comment:
I think this won't work with an iterable (because you need to iterate over the iterable in order to know whether it is empty).
Maybe in this case we could just remove the logging?